scripts/ci-conformance.sh

```diff
@@ -50,7 +50,7 @@ get_random_region() {
     echo "${REGIONS[${RANDOM} % ${#REGIONS[@]}]}"
 }
 
-export LOCAL_ONLY=${LOCAL_ONLY:-"false"}
+export LOCAL_ONLY=${LOCAL_ONLY:-"true"}
```

Comment:
@chewong you might need to update any jobs that need to push conformance artifacts to an outside registry to have LOCAL_ONLY set to true.

Reply:
https://github.com/kubernetes/test-infra/pull/21953